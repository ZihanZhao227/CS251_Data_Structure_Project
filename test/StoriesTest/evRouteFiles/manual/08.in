60
0 0 0 CITY Start
1 38 89 CITY CITY_1
2 86 122 CHARGING_STATION CHARGING_STATION_2
3 27 109 RESTAURANT RESTAURANT_3
4 94 164 CHARGING_STATION CHARGING_STATION_4
5 146 120 CHARGING_STATION CHARGING_STATION_5
6 103 46 CHARGING_STATION CHARGING_STATION_6
7 21 128 TOURIST_ATTRACTION TOURIST_ATTRACTION_7
8 190 8 CHARGING_STATION CHARGING_STATION_8
9 52 144 RESTAURANT RESTAURANT_9
10 105 99 CHARGING_STATION CHARGING_STATION_10
11 13 191 RESTAURANT RESTAURANT_11
12 178 41 RESTAURANT RESTAURANT_12
13 111 195 CHARGING_STATION CHARGING_STATION_13
14 172 157 CHARGING_STATION CHARGING_STATION_14
15 191 178 CHARGING_STATION CHARGING_STATION_15
16 50 50 CHARGING_STATION CHARGING_STATION_16
17 179 147 CITY CITY_17
18 190 44 CHARGING_STATION CHARGING_STATION_18
19 110 177 CITY CITY_19
20 45 66 CHARGING_STATION CHARGING_STATION_20
21 27 35 CITY CITY_21
22 72 143 CITY CITY_22
23 27 59 CHARGING_STATION CHARGING_STATION_23
24 39 148 CHARGING_STATION CHARGING_STATION_24
25 126 181 RESTAURANT RESTAURANT_25
26 122 9 CITY CITY_26
27 12 193 TOURIST_ATTRACTION TOURIST_ATTRACTION_27
28 74 97 CITY CITY_28
29 62 146 CHARGING_STATION CHARGING_STATION_29
30 118 105 CHARGING_STATION CHARGING_STATION_30
31 96 131 CHARGING_STATION CHARGING_STATION_31
32 110 131 RESTAURANT RESTAURANT_32
33 90 71 CHARGING_STATION CHARGING_STATION_33
34 184 119 CHARGING_STATION CHARGING_STATION_34
35 94 192 TOURIST_ATTRACTION TOURIST_ATTRACTION_35
36 24 97 RESTAURANT RESTAURANT_36
37 55 105 CHARGING_STATION CHARGING_STATION_37
38 94 173 CHARGING_STATION CHARGING_STATION_38
39 97 153 CHARGING_STATION CHARGING_STATION_39
40 38 182 CITY CITY_40
41 94 179 REGULAR_WAYPOINT REGULAR_WAYPOINT_41
42 136 92 RESTAURANT RESTAURANT_42
43 25 92 CITY CITY_43
44 27 106 CITY CITY_44
45 190 84 TOURIST_ATTRACTION TOURIST_ATTRACTION_45
46 121 116 REGULAR_WAYPOINT REGULAR_WAYPOINT_46
47 106 118 CITY CITY_47
48 147 79 CHARGING_STATION CHARGING_STATION_48
49 187 11 CITY CITY_49
50 101 191 CHARGING_STATION CHARGING_STATION_50
51 131 133 CHARGING_STATION CHARGING_STATION_51
52 139 105 CHARGING_STATION CHARGING_STATION_52
53 179 92 CHARGING_STATION CHARGING_STATION_53
54 28 0 REGULAR_WAYPOINT REGULAR_WAYPOINT_54
55 154 191 TOURIST_ATTRACTION TOURIST_ATTRACTION_55
56 59 66 TOURIST_ATTRACTION TOURIST_ATTRACTION_56
57 114 39 CHARGING_STATION CHARGING_STATION_57
58 169 49 TOURIST_ATTRACTION TOURIST_ATTRACTION_58
59 200 200 CITY End
