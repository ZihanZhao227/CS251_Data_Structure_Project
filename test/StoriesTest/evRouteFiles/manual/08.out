0 59 90.000 284.302
0 16 10 14 59
